src/System.IO.FileSystem/tests/File/GetSetAttributes.cs

```diff
@@ -101,5 +101,40 @@ public void WindowsInvalidAttributes(FileAttributes attr)
             Set(path, attr);
             Assert.Equal(FileAttributes.Normal, Get(path));
         }
+
+        [SkippableFact]
+        public void SymLinksAreReparsePoints()
+        {
+            var path = GetTestFilePath();
+            var linkPath = GetTestFilePath();
+            File.Create(path).Dispose();
```

Comment:
Makes sense. I used this syntax because it's the same as how empty temp files are created elsewhere in these tests. If you think it's better to use TempFile, though, I can make the switch.